packages/lab/src/stepper-input/useStepperInput.ts

```diff
@@ -1,17 +1,13 @@
-import { KeyboardEvent, MouseEvent, MutableRefObject } from "react";
-import { ButtonProps, useControlled, useId } from "@salt-ds/core";
-import { InputLegacyProps as InputProps } from "../input-legacy";
-import { useDynamicAriaLabel } from "./internal/useDynamicAriaLabel";
+import {
+  ChangeEvent,
+  KeyboardEvent,
+  MouseEvent,
+  MutableRefObject,
+} from "react";
+import { useControlled, useId, InputProps } from "@salt-ds/core";
 import { useSpinner } from "./internal/useSpinner";
 import { StepperInputProps } from "./StepperInput";
 
-type Direction = "decrement" | "increment";
-
-const stepperDirection = {
-  INCREMENT: "increment" as Direction,
-  DECREMENT: "decrement" as Direction,
-};
-
 // The input should only accept numbers, decimal points, and plus/minus symbols
 const ACCEPT_INPUT = /^[-+]?[0-9]*\.?([0-9]+)?/g;
```

Comment:
If we explicitly not allowing `e` (which is possible in `<input type="number"/>`), need to document it somehow

Reply:
will follow up about this

Reply:
what was the follow up here... there is a danger that we will lose that follow up, unless we capture the action somewhere ?